Curve-finance-suite/src/tests/unkill.spec.ts

```diff
@@ -0,0 +1,44 @@
+import {
+  Finding,
+  HandleTransaction,
+  createTransactionEvent,
+} from "forta-agent";
+import { createAddress } from "nethermindeth-general-agents-module";
+import provideUnkillAgent, { web3, unkill } from "../agents/unkill";
+
+const ADDRESS = createAddress("11");
```

Comment:
`createAddress` won't add the `0x` prefix for you, maybe you should add it. It would look like `createAddress("0x11")`